src/pages/home/recepie.tsx

```diff
@@ -0,0 +1,4 @@
+function DisplayRecepie() {
+  return <div>This is the recepie page</div>;
```

Comment:
this works! but I think a variant of this message that includes the id (pulled in from url param) will be better :D - that way developer will get an instant feedback from this skeleton route.

an example would be like: `Displaying recipe details for : id` as mentioned in the task